docs/Reference/Certificates/generateProvisioningCert.md

```diff
@@ -0,0 +1,135 @@
+--8<-- "References/abbreviations.md"
+	
+## Why generate a customer provisioning certifcate?
+
+The importance of generating a provisioning certificate for customer use is due to the fact that it provides a way to autheticate the device. In essence it is an extra layer of security. 
+
+## How to generate a customer provisioning certificate? 
+
+To begin generating a provisioning certificate the open source command line tool, OpenSSL, is required. OpenSSL can be used in areas of cryptography, enctyption, and digital signatures. 
+
+If you have already gone through and downloaded the necessary prequisites from the ["Getting Started Guide"](https://open-amt-cloud-toolkit.github.io/docs/2.13/GetStarted/prerequisites/), specifically "git*", then all you have to do is search for "git bash" on your machine and open the terminal and you will be able to access OpenSSL from there. 
+
+You will also need to prepare two files, cert.conf and csr.conf file.  
+
+**cert.conf**: This is the certificate configuration file. It is used primarily for to define specific settings for the certificate. 
+
+!!! example file
+    ```
+    basicConstraints = CA:TRUE
+    subjectKeyIdentifier = random
+    authorityKeyIdentifier = keyid,issuer:always
+    keyUsage = nonRepudiation, cRLSign
+    extendedKeyUsage = clientAuth, 1.2.3.4.5.6
+    subjectAltName = @
+    alt_names
+    
+    [alt_names]
+    DNS.1 = example.domain
+    DNS.2 = test.domain
+    IP.1 = 192.168.1.1
+    IP.2 = 10.0.0.1
+    ```
+**csr.conf**: This is the certificate signing request configuration file. It is meant for information related to identification . 
+
+!!! example file
+    ```
+    [ req ]
+    default_bits = 2048
+    prompt = no
+    default_md = sha256
+    distinguished_name = dn
+
+    [ dn ]
+    C=US
+    ST=California
+    L=Example City
+    O=Example Organization
+    OU=IT Department
+    CN=www.example.com
```

Comment:
Add a comment for users to change the CN (www.example.com) to their server's FQDN. In our case, it is vprodemo.com